### PR TITLE
1146: Created the pages using materail ui and emotion as per the Figm…

### PR DESCRIPTION
**Pull request by md-huzaifa**
…a for the assignment

### CODE REVIEW

File: src/domains/page2/Page2.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            </FormControl>
+            <FormControl fullWidth sx={{ m: 1 }}>
+              <InputLabel htmlFor="outlined-adornment">label</InputLabel>
+              <OutlinedInput
+                id="outlined-adornment"
+                endAdornment={
+                  <InputAdornment position="end">
+                    <CancelOutlinedIcon />
+                  </InputAdornment>
+                }
```
Comment (atmeshwar-classic) [inline src/domains/page2/Page2.tsx:59]: This doesn't seem to be working. Please check

File: src/routes/routes.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Route, Routes } from "react-router-dom";
+import Page1 from "../domains/page1/Page1";
+import Page2 from "../domains/page2/Page2";
+import Page3 from "../domains/page3/Page3";
+
+const Routings = () => (
```
Comment (atmeshwar-classic) [inline src/routes/routes.tsx:6]: Just name it Router, Routings is not a real word.

File: src/components/NavBar/NavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { AppBar, Box, Button, Toolbar, useTheme } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+const NavBar: React.FC = () => {
+  const theme = useTheme();
+  const navigate = useNavigate();
+  return (
+    <AppBar
+      sx={{backgroundColor:'white', display: "flex", alignItems: "flex-end" }}
```
Comment (atmeshwar-classic) [inline src/components/NavBar/NavBar.tsx:9]: Using "sx" to write styles is good for a small application. But, it is not scalable.
MUI docs suggest using "Emotion" lib to handle styling.

**Review by atmeshwar-classic — CHANGES_REQUESTED** [review]
Hi @md-huzaifa - It looks good! Just a few things if you can fix.

File: src/components/NavBar/NavBar.tsx (hunk 3 above)
Comment (md-huzaifa) [reply src/components/NavBar/NavBar.tsx]: @atmeshwar-classic  Updated the changes can you please review them

File: src/Styles/style.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  marginRight: "5px",
+  padding: "10px",
+  "&:hover": {
+    backgroundColor: "#FFFFFF",
+    color: "#6750A4",
+    border: "2px solid",
+    borderRadius: "30px",
+    borderColor: "#EADDFF",
+  },
+});
```
Comment (atmeshwar-classic) [inline src/Styles/style.ts:26]: The previous buttons were fine. This was not needed.
Only use it when you have a very custom design.
In our requirement, we only needed to change the colors which was handled by the theme itself.

Also, don't name them 'StyledButton'. Use 'Button' or if it's a custom button then name it according to its use case.

**Review by atmeshwar-classic — APPROVED** [review]
Hi @md-huzaifa - Looks good. Please look at the one comment I have made.

File: src/Styles/style.ts (hunk 4 above)
Comment (md-huzaifa) [reply src/Styles/style.ts:26]: Okay Atmeshwar. I used this Buttons as I had to use sx props everywhere I need Button from mui.I will change the name of these Buttons accordingly

**Comment by atmeshwar-classic** [comment]
Closing

**Pull request closed without merging**